node-server.js

```diff
@@ -37,7 +37,7 @@ function askQuestions () {
       type: 'input',
       name: 'client_id',
       message: 'Client ID: ',
-      validate: input => /^\s*[a-zA-Z0-9_-]{14}\s*$/.test(input) || 'Please enter your client ID. This is the 14-character string that appears to the right of your app\'s icon on this page: https://www.reddit.com/prefs/apps',
+      validate: input => /^\s*[a-zA-Z0-9_-]{22}\s*$/.test(input) || 'Please enter your client ID. This is the 22-character string that appears to the right of your app\'s icon on this page: https://www.reddit.com/prefs/apps',
```

Comment:
Thanks for the pull request! I think client IDs were 14 characters in the past, but reddit must have changed how it generates IDs. As a result, I don't think the CLI should require that client IDs are 22 characters, since some users might still have apps with legacy 14-character client IDs.

Instead, I think it's fine to be lenient about the ID length and allow anything that's 14 characters or more:

```suggestion
      validate: input => /^\s*[a-zA-Z0-9_-]{14,}\s*$/.test(input) || 'Please enter your client ID. This is the random-looking string that appears to the right of your app\'s icon on this page: https://www.reddit.com/prefs/apps',
```